### PR TITLE
Add the `osu!monthly, September 2022` news post

### PR DESCRIPTION
**Pull request by 0x84f**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: news/2022-10-01-osumonthly-3.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+- The [osu! university](https://discord.com/invite/QubdHdnBVg) rises from its ashes with the [osu! improvement month](https://osu.ppy.sh/community/forums/topics/1649135?n=1) event that stretches over the whole month of October. Interviews, tournaments, coaching, and giveaways are planned, and it would definitely be a mistake not to take advantage of this opportunity to socialize with the community, or to follow the events of your favorite game!
+- After several weeks of hard work by creators and the voting phase coming and going, we should soon be able to see the results of the [triangles Beatmapping Contest](/wiki/Contests/triangles_Beatmapping_Contest). Who was able to combine the best storyboard, hitsound, and mapping around the triangles theme? We can't wait to see!
+- The voting stage for the [Chromatic Alteration skinning contest](https://osu.ppy.sh/home/news/2022-07-18-skinning-contest-chromatic-alteration-announcement#challenge:-chromatic-alteration) has concluded! With 23 entries and only 3 votes per player, hard decisions had to be made when it came to figuring out which entries deserve votes the most, and we are all eagerly awaiting to see the final results!
```
Comment (RockRoller01) [inline news/2022-10-01-osumonthly-3.md:158]: dont see any reason why its linking to the challenge section, should probably just link to the newspost without the #id

**Comment by 0x84f** [comment]
@Ephemeralis as discussed previously you should check the wording for the Corsace part

**Review by Walavouchey — DISMISSED** [review]
just the quiz question left

**Comment by peppy** [comment]
Is this planned to go out soon?

**Comment by Walavouchey** [comment]
[about today](https://discord.com/channels/188630481301012481/1003689172420276246/1025428815196995634) (@Ephemeralis)

**Review by Walavouchey — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**